snap/hooks/configure

```diff
@@ -12,6 +12,27 @@ EOT
 let changes+=1
 fi
 
+# display-config
+display_config=$(snapctl get display-config)
+if [ -n "${display_config}" ]; then
```

Comment:
Hmm so if we set it to empty, the file will remain, along with the "DO NOT EDIT" header... Maybe we could check if the file was written by us (i.e. has the `# DO NOT EDIT` header) and remove it if that's the case?